content/blog/entries/cc-vocabulary-docs-updates-closing/contents.lr

```diff
@@ -0,0 +1,237 @@
+- https://github.com/creativecommons/vocabulary/pull/806 - Dark mode for our storybooks
+
+<center>
+<img alt"Dark Mode" src="darkmode.png"/><br>
+<small class="muted">Behold, the dark theme!</small>
+</center>
+
+- https://github.com/creativecommons/vocabulary/pull/802 - [Grid Documentation Story](https://cc-vocabulary.netlify.app/?path=/docs/layouts-grid--fullhd#grid-system)
+
+<center>
+<img alt"Grid Docs" src="grid.png"/><br>
+<small class="muted">Grid Documenation Story</small>
+</center>
+
+- https://github.com/creativecommons/vocabulary/pull/785 - [Monorepo Document Story](https://cc-vocabulary.netlify.app/?path=/docs/vocabulary-structure--page#why-is-vocabulary-a-monorepo)
+
+<center>
+<img alt"Monorepo Document Story" src="structure.png"/><br>
+<small class="muted">Monorepo Structure Story</small>
+</center>
+
+- https://github.com/creativecommons/vocabulary/pull/774 - [Getting Started Story](https://cc-vocabulary.netlify.app/?path=/story/vocabulary-getting-started--page#getting-started)
+
+- https://github.com/creativecommons/vocabulary/pull/734 - README.md customisation
+- https://github.com/creativecommons/vocabulary/pull/671 - Add a CHANGELOG.md to fit conventions
+- https://github.com/creativecommons/vocabulary/pull/649 - Unified README.md and updated build process
+- https://github.com/creativecommons/vocabulary/pull/452 - Configured native dependabot
+- https://github.com/creativecommons/vocabulary/pull/445 - Phone Screen backgrounds
+- https://github.com/creativecommons/vocabulary/issues/757 - Snapshot testing on Vocabulary
+- https://github.com/creativecommons/vocabulary/issues/735 - Document Component Capable of rendering markdown
+- https://github.com/creativecommons/vocabulary/issues/807 - Add a Maintained with lerna badge to README.md
+- https://github.com/creativecommons/vocabulary/issues/776 - Add a new Install size badge
+- https://github.com/creativecommons/vocabulary/issues/736 - Customize npm README.md for individual packages
+
+  
+On the [creativecommons/creativecommons.github.io-source](https://github.com/creativecommons/creativecommons.github.io-source) repository:
+- https://github.com/creativecommons/creativecommons.github.io-source/pull/530 - Introductory first blog post
+- https://github.com/creativecommons/creativecommons.github.io-source/pull/549 - Vocabulary Site Update v1
+- https://github.com/creativecommons/creativecommons.github.io-source/pull/555 - Vocabulary Site Mid-Internship Update v2
+- https://github.com/creativecommons/creativecommons.github.io-source/pull/561 - Vocabulary Site Update v3
+
+On the [creativecommons/ccos-scripts](https://github.com/creativecommons/ccos-scripts) repository:
+
+- https://github.com/creativecommons/ccos-scripts/pull/100 - Fix file extension in README.md
+
+On the [creativecommons/cccatalog-api](https://github.com/creativecommons/cccatalog-api) repository:
+
+- https://github.com/creativecommons/cccatalog-api/pull/533 - Configured Dependabot
```

Comment:
I think a table would be more suitable for this^?

Reply:
Yes, now that you mentioned it I think so too! 👍